### ISSUE TITLE
 Feature Request: Node to modify the "dependencies" input/output

### ISSUE THREAD

**Issue by diffustar** [jags111/efficiency-nodes-comfyui#158]
There currently seems to be no way to modify the dependencies input required for certain XY plot types.

Think it would be nice if there was a node that takes a base "dependencies" as input together with optional extra keys  to output the mixed dependencies. Something like that maybe ?

**Comment by Fittiboy** [comment]
Agreed, this would be useful! I handle LoRAs and conditioning outside the efficient loader, and the dependencies then override all of that when I want to create an X/Y plot. It would be great to have a node that just expands the dependencies into its components, like pipe nodes.

**Comment by klromans557** [comment]
> Agreed, this would be useful! I handle LoRAs and conditioning outside the efficient loader, and the dependencies then override all of that when I want to create an X/Y plot. It would be great to have a node that just expands the dependencies into its components, like pipe nodes.

I keep hitting this wall, and this simple pipe-esque dependency node would be so helpful. :D
